### PR TITLE
add es curator link

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Closes https://github.com/camunda/camunda-docs/issues/3225.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by akeller** [comment]
I'm not sure I can approve this without some help from @marcosgvieira. This page has me confused - https://docs.camunda.io/docs/self-managed/operational-guides/update-guide/elasticsearch/7-to-8/#elasticsearch-8-curator

When we link to the ES curator, do we need to link to a specific version?

**Comment by akeller** [comment]
@christinaausley please follow up on this on Slack with the Tasklist team. See my comment above regarding ES versions.

**Comment by christinaausley** [comment]
According to @houssain-barouni, this depends on the version of Elasticsearch being used.
- If they use ES7, ES Curator should be used. However, it seems that ES7 supports ILM policy as well https://www.elastic.co/guide/en/elasticsearch/reference/7.17/index-lifecycle-management.html.
- If they use ES8, ILM should be used.  ILM is supported as of 8.2, and can be enabled by configuring alongside https://docs.camunda.io/docs/self-managed/tasklist-deployment/data-retention/#snippet-from-applicationyml.

**Comment by houssain-barouni** [comment]
@christinaausley I think we can just talk about [ILM policies](https://www.elastic.co/guide/en/elasticsearch/reference/8.12/index-lifecycle-management.html) as it is supported by both ES7 and ES8, and supported by Tasklist since 8.2 
Customers still using curators can rely on the next paragraph if they plan to move to ES8

**Comment by christinaausley** [comment]
> We should add a link to the [ilmEnabled configuration](https://docs.camunda.io/docs/self-managed/tasklist-deployment/data-retention/#snippet-from-applicationyml). For customers using Helm charts, it is ON by default since 8.3

@houssain-barouni This is noted within the data retention page, do you want to see it linked elsewhere?

**Comment by christinaausley** [comment]
@houssain-barouni Thoughts on the rewording across versions?

File: docs/self-managed/tasklist-deployment/data-retention.md [hunk 1]
```diff
@@ -31,9 +31,9 @@ The default time between a process instance finishing and being moved to a dated
...
 
-Dated indices may be safely removed from Elasticsearch. "Safely" means only finished process instances are deleted together with all related data, and the rest of the data stays consistent. You can use Elasticsearch Curator or other tools/scripts to delete old data.
+Dated indices may be safely removed from Elasticsearch. "Safely" means only finished process instances are deleted together with all related data, and the rest of the data stays consistent.
 
-Users updating from Elasticsearch 7 to Elasticsearch 8 will encounter issues with the Elasticsearch Curator. To resolve this, Tasklist allows configuring an Index Lifecycle Management (ILM) Policy using the `archiver` configuration options:
+Users updating from Elasticsearch 7 to Elasticsearch 8 will encounter issues with the [Elasticsearch Curator](https://www.elastic.co/guide/en/elasticsearch/client/curator/current/index.html). To resolve this, Tasklist allows configuring an Index Lifecycle Management (ILM) Policy using the `archiver` configuration options, which is enabled by default:
```
Comment (akeller) [inline docs/self-managed/tasklist-deployment/data-retention.md:36]: Can we link to this page we maintain in our docs as well? https://docs.camunda.io/docs/self-managed/operational-guides/update-guide/elasticsearch/7-to-8/

We may need to float some of this to the **Update guide** as well. I'm ok with this happening in a separate PR scope.

**Comment by akeller** [comment]
@christinaausley are you looking for a final review here or still working on this?

**Review by akeller — APPROVED** [review]

**Pull request merged**